layers/decompression/shaders/spirv/GInflate16_HAVE_INT16_HAVE_INT64_vk.h

```diff
@@ -0,0 +1,925 @@
+// 1112.0.0
```

Comment:
still need copyright headers and "this is generated code" comments on these files. Why do all of the files have 1112.0.0 as their only comment? Is it a version number for something?

Reply:
These files are generated when glslangValidator is run on a GLSL file and converted to SPIRV header. The SPIRV is generated in a header form that can be included in a CPP file with this option:

 --vn <name>                       creates a C header file that contains a
                                    uint32_t array named <name>
                                    initialized with the shader binary code

Since these files are just spitted out of glslangValidator, we would need additional scripting to add copyright/generated code info. Would adding copyright to the generator (i.e. compile.sh/compile.bat) suffice?

Yes, 1112.0.0 is the output from glslangValidator and I assume it is some form of revisioning.

Reply:
I think it makes sense to add copyright to the actual source file(GLSL) and the generator, but not sure about adding it to the output of a commandline tool. I will let you decide this though. Thanks!

Reply:
Just noticed this now, it would have saved you trouble if I had replied sooner... but for all of the existing generated code in this repository and Vulkan-ValidationLayers we do have copyright headers.